shared/templates/c_file_structure.txt

```diff
@@ -1,54 +1,43 @@
-include
```

Comment:
why delete these lol? makes it easier to search for the one you're looking for in the file

Reply:
wasnt thinking about that, always did a goto line but fair point. removed because easier when copying chunks in for a new file.
Have reverted